src/jit/bitsetasshortlong.h

```diff
@@ -664,9 +664,11 @@ void BitSetOps</*BitSetType*/ BitSetShortLongRep,
                /*BitSetTraits*/ BitSetTraits>::ClearDLong(Env env, BitSetShortLongRep& bs)
 {
     assert(!IsShort(env));
-    // Recall that ClearD does *not* require "bs" to be of the current epoch.
-    // Therefore, we must allocate a new representation.
-    bs = MakeEmptyArrayBits(env);
```

Comment:
@sandreenko We had talked about you converting existing ClearD() callers to call MakeEmpty() directly, for existing callers that could not easily be proven to work with the new semantics. I presume the existing semantics, and huge comment you deleted for ClearD(), were there for a reason.

Is there some reason you decided not to do that? Could you somehow convince yourself the new semantics work for every call site?